cli/lib/commands/whoami.js

```diff
@@ -29,12 +38,13 @@ async function whoami(opts) {
   }
 
   if (response.status > 399) {
-    opts.log.error(body.message || body);
+    opts.log.error(body.message || body.error || body);
```

Comment:
ooh, good catch. if we're using `body.error` someplace that's a mistake on the server side – we should fix that.